newsroom/design.py

```diff
@@ -1,14 +1,25 @@
+from typing import Optional
+from pydantic import BaseModel
 from datetime import datetime, timedelta
-from superdesk.flask import Blueprint, url_for, render_template
 
-blueprint = Blueprint("design", __name__)
+from superdesk.flask import url_for, render_template
+from superdesk.core.web import EndpointGroup
+from superdesk.core.types import Request
+from superdesk.core.module import Module
 
+blueprint = EndpointGroup("design", __name__)
 
-@blueprint.route("/design/detail")
-async def detail():
+
+class RouteArguments(BaseModel):
+    page: Optional[str] = None
```

Comment:
Following above suggestion
```suggestion
    page: str | None = None
```